### DISCUSSION TITLE
Вопросы по docker

### DISCUSSION THREAD

**Discussion by napaster** [mikopbx/Core#423] [Q&A]
Поставил чистую систему centos 8 stream, без графики без всего. В ней запустил docker. Есть пара вопросов по документации. 
У вас в мануале написано 
```# Вариант запуска MikoPBX
# НЕ привилегированный режим. Пользователь и группа "www-data" дожены существовать в системе:
docker run --cap-add=NET_ADMIN --net=host --name mikopbx \
           -v /var/spool/mikopbx/cf:/cf \
           -v /var/spool/mikopbx/storage:/storage \
           -e SSH_PORT=23 \
           -e ID_WWW_USER="$(id -u www-data)" \
           -e ID_WWW_GROUP="$(id -g www-data)" \
           -it -d --restart always ghcr.io/mikopbx/mikopbx-x86-64
```
Вопрос а если у меня нету этого пользователя и группы ? как быть тогда ? это обязательная опция или нет ? и на что влиеяет ?

**Comment by boffart** [comment]
Да это обязательная опция, если нет пользователя, то создайте нового с ограниченными правами. 
Без этих опций работать корректно не будет.

**Comment by napaster** [comment]
Понял. Спасибо за ответ.
